configure.ac

```diff
@@ -322,7 +322,7 @@ show_section_title "Check for functions, types and structures"
 
 # Check fortified glibc functions
 #
```

Comment:
This still doesn't work correctly for me. Even in a static build, it decides that these functions exist here. But when I compile the static library I get errors like this:

sock/sock-redirect.cpp: In function ‘ssize_t xlio___read_chk(int, void*, size_t, size_t)’:
sock/sock-redirect.cpp:1365:20: error: ‘::__read_chk’ has not been declared
 1365 |     return SYSCALL(__read_chk, __fd, __buf, __nbytes, __buflen);

Reply:
Will enable the functions for non-static build